modules/navier_stokes/src/loops/GatherRCDataElementThread.C

```diff
@@ -48,7 +49,14 @@ GatherRCDataElementThread::subdomainChanged()
     copied_queries.template condition<AttribVar>(static_cast<int>(var_num)).queryInto(var_eks);
     for (auto var_ek : var_eks)
       if (auto insfv_ek = dynamic_cast<INSFVMomentumResidualObject *>(var_ek))
+      {
+        // On INITIAL, the time integrator is not ready for these residual objects
```

Comment:
So... if you are not executing VFRs on internal boundaries, then can't you simply not execute the RC user object on initial? E.g. can we just keep carrying on with the RC UO only executing on `EXEC_PRE_KERNELS`? Then we don't have to do this stuff here

Reply:
oh I forgot to delete that code. Good catch

I removed the EXEC_INITIAL from the RC UO already